### PR TITLE
CI/CD setup

### PR DESCRIPTION
**Pull request by Muneerah1612**
## I created the workflow for Continuous Integration using GitHub actions, added Procfile to enable deployment to Heroku

P.S: Build will fail, because Secret Key has not been added to GitHub

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,3 +0,0 @@
-SECRET_KEY=somevalue
-USER=somevalue
-PASSWORD=somevalue
```
Comment (eguonoefe) [inline .env.sample:3]: Why was this file deleted?

**Review by eguonoefe — CHANGES_REQUESTED** [review]

File: .env.sample (hunk 1 above)
Comment (Muneerah1612) [reply .env.sample:3]: I thought it was not necessary since we all have our .env file already

Comment (eguonoefe) [reply .env.sample:3]: Okay. So what happens if someone who is new to the project clones the project?

Comment (eguonoefe) [reply .env.sample:3]: How do they know what to put in the .env file?

Comment (Muneerah1612) [reply .env.sample:3]: Oh, I did not think about that. I will add it back now.

**Review by eguonoefe — APPROVED** [review]
Looks good to me

**Review by OreLana — APPROVED** [review]
Looks good

**Pull request merged**